arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/call/CallSection.java

```diff
@@ -145,6 +146,7 @@ public CallSection(Hub hub) {
 
     // MXPX case
     if (Exceptions.memoryExpansionException(exceptions)) {
+      commonValues.setTracedException(TracedException.MEMORY_EXPANSION_EXCEPTION);
```

Comment:
After re-reading this I believe we can include the `CALL` family in the high level `TracedException` setting.